src/util-thash.c

```diff
@@ -173,8 +171,7 @@ static THashData *THashDataAlloc(THashTableContext *ctx)
 
     /* points to data right after THashData block */
     h->data = (uint8_t *)h + sizeof(THashData);
-
-//    memset(h, 0x00, data_size);
+    //    memset(h, 0x00, data_size);
```

Comment:
why do we keep this comment?

Reply:
I'm not sure. I'm not the original author of this part so I let it be. BUT, this should not have been modified by clang so it means I touched it somehow after all..